src/App.tsx

```diff
@@ -1,24 +1,199 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { TodoHeader } from './components/TodoHeader';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+
+import { ErrorOnPage } from './components/ErrorOnPage';
+
+import { FilterBy } from './utils/FilterBy';
+import { Todo } from './types/Todo';
+import {
+  getTodos, createTodo, removeTodo, updateTodo,
+} from './todos';
+
+import { getFilteredTodos } from './utils/NewfilterTodos';
+
+import { ErrorMessages } from './types/ErrorNessages';
+import { client } from './utils/fetchClient';
+
+const USER_ID = 1333;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(FilterBy.All);
+  const [newError, setNewError] = useState<ErrorMessages | null>(null);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [deleteTodosId, setDeleteTodosId] = useState<number[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
```

Comment:
Rename boolean variables using this [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(true);
```